src/sensor/ping360.cpp

```diff
@@ -73,6 +73,20 @@ Ping360::Ping360()
 
 void Ping360::startPreConfigurationProcess()
 {
+    // Stop all configuration/message timers if link is not writable
```

Comment:
is this for replaying logs? What kinds of links are not writeable?

Reply:
yes, log is also non writable.